examples/function_memory_game/src/components/chessboard.rs

```diff
@@ -0,0 +1,24 @@
+use yew::prelude::*;
+use yew::{function_component, html, Properties};
+
+use crate::components::chessboard_card::ChessboardCard;
+use crate::state::{Card, RawCard};
+
+#[derive(Properties, Clone, PartialEq)]
+pub struct Props {
+    pub cards: Vec<Card>,
+    pub on_flip: Callback<RawCard>,
+}
+
+#[function_component(Chessboard)]
+pub fn chessboard(props: &Props) -> Html {
+    html! {
+        <div class="chess-board">
+        { for props.cards.iter().map(|card|
+            html! {
+                <ChessboardCard card={card.clone()} on_flip={&props.on_flip} />
+            }
+        ) }
+        </div>
+    }
+}
```

Comment:
```suggestion
#[derive(Properties, Clone, PartialEq)]
pub struct Props {
    pub cards: Vec<Card>,
    pub on_flip: Callback<RawCard>,
}

#[function_component]
pub fn Chessboard(props: &Props) -> Html {
    html! {
        <div class="chess-board">
        { for props.cards.into_iter().map(|card|
            html! {
                <ChessboardCard {card} on_flip={&props.on_flip} />
            }
        ) }
        </div>
    }
}
```

Reply:
I don't think you can call `into_iter` on a `&Vec<_>`

Reply:
@hamza1311 true